V0/components/onboarding-analytics-dashboard.tsx

```
@@ -0,0 +1,646 @@
+        )
+      })}
+    </div>
+  )
+}
+
+interface ABTestResultsProps {
+  tests: ABTest[]
+  results: Map<string, ABTestResults[]>
+}
+
+const ABTestResults: React.FC<ABTestResultsProps> = ({ tests, results }) => {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'ABTestResults'. Consider to delete it or rename it.

'ABTestResults' is defined here:


(lint/suspicious/noRedeclare)